### ISSUE TITLE
Message Customization

### ISSUE THREAD

**Issue by lokka30** [JEFF-Media-GbR/Spigot-UpdateChecker#20]
Hi @mfnalex, should there be a way for plugins to call a method to change the text displayed by the update checker?

Advantages:
* can have adjustable update checker messages in the plugin configuration (e.g. `messages.yml`)
* can customize the colors

Would like to hear what you think :)

**Comment by lokka30** [comment]
Just discovered #9 right after posting this issue. However, this issue still has a greater scope than just customizing player messages.

**Comment by mfnalex** [comment]
Yes, I already planned on adding that, but I was too lazy by now lol. Feel free to PR :) Otherwise I'll do it someday(TM)

**Comment by lokka30** [comment]
I hope to be able to make a few PRs (also someday™️ 😂), however, uni does not make that easy 😬
